### ISSUE TITLE
Remove live search bar and accordion nav

### ISSUE THREAD

**Issue by mattheu** [humanmade/hm-pattern-library#124]
The live search bar and the accordion nav were built for the handbook, but I think they should be part of that theme and need included in the pattern library. 

- They wont be useful for everyone using this project.
- Both add complexity, particularly with regard to the JS. 
- The design don't really fit with the new design so well and need some attention. There isn't much in the new design that is similar and we can steal from :)

So the simplest fix is to just remove :) Remember major releases do not maintain backwards compatibility so this isn't a problem.

Basically I want to ship a new version of the pattern library and having to overhaul these is a big blocker.

**Comment by sambulance** [comment]
This was merged
